### DISCUSSION TITLE
No way to hode toc?

### DISCUSSION THREAD

**Discussion by vitaly-zdanevich** [facebook/docusaurus#8071] [Q&A]
For example [here](https://jestjs.io/docs/expect#tomatchobjectobject) 
![image](https://user-images.githubusercontent.com/3514015/189217305-cf106d46-58e1-48b2-b1b4-8c38e021c208.png)

No way to hide left/right sidebars?

I have already [asked on their repo](https://github.com/facebook/jest/issues/13228).

**Comment by slorber** [comment]
the left sidebar can be collasped, the right toc can be hidden with frontmatter

Not sure to understand what's your point here

**Reply by pelikhan** [reply]
docs here: https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-docs#hide_table_of_contents
